keyboards/planck/keymaps/copface/rules.mk

```diff
@@ -0,0 +1,11 @@
+SRC += muse.c
+
+BOOTMAGIC_ENABLE = no
```

Comment:
You sure about this?  
Having at least `lite` available can come in handy.

Reply:
I guess I've never had to use any of the functionality ever.  
Could you give me some examples of when this would come in handy? I would greatly appreciate it.

Reply:
It's just a +1 method of booting into bootloader without disassembling your board.  

Anyway, it was just a suggestion.

Reply:
Thanks for the suggestion.

Is booting into bootloader any different than pressing the reset button that comes on plancks or pressing the key combo I have for getting into that same mode?